cmake/gt_bindings.cmake.in

```diff
@@ -52,9 +54,11 @@ target_link_libraries(c_bindings_generator PRIVATE Boost::boost) #TODO make an i
 # However if the user wants to use the target at a later stage, e.g. in testing (with Fortran enabled), the target can
 # be created by a call to gt_enable_bindings_library_fortran().
 macro(gt_enable_bindings_library_fortran target_name)
+    set(__C_BINDINGS_SOURCE_DIR @__C_BINDINGS_SOURCE_DIR@)
```

Comment:
This could also be a function I think, then we don't need unset, right?

Reply:
Yes, I expected that it is a macro for a reason. I should have checked that...